src/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { combineReducers } from 'redux';
+import book from './books/books';
+import category from './categories/categories';
+
+export default combineReducers({
+  book,
+  category,
+});
```

Comment:
- [ ] It better to save combined reducers in store variable, and export it. 
- [ ] You are combining reducers but not creating a store, 
- [ ] OPTIONAL: I noticed that you called one reducers as Singular items (book, category), I strongly recommend to name in plural, because your `book` reducer is supposed to hold all books and it it would make scene to have the reducer in plural form

Here is an example of your configStore: 
```
import { combineReducers, createStore } from 'react-redux';

const rootReducer = combineReducers({
  categories,
  books,
});

const store = createStore(rootReducer);

export default store;
```